### PR TITLE
RX/TX, uptime and temperature + RUT240

### PR DESCRIPTION
**Pull request by KEGustafsson**
Would you consider following changes?
1) Usage data is splitted to RX and TX paths, which allow easier monitoring of down/upload e.g. in Grafana.
2) Modem uptime and temperature readings are added to monitor health of the modem.
3) Support following modems: RUT240, 360, 950, 955, X9, X11, X14. RUT240 selector added due to different registers for data usage.
4) Name fields changed to more generic

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -56,6 +61,11 @@ module.exports = function createPlugin(app) {
           path: 'networking.lte.radioQuality',
           value: radioQuality,
         });
+        const modemTemperature = Buffer.concat(data.slice(4, 7)).readInt32BE() / 10;
+        values.push({
+          path: 'networking.modem.temperature',
```
Comment (bergie) [inline index.js:66]: Temperatures in Signal K should be in Kelvin, and are typically under `environment`. Can you change that?

**Review by bergie — CHANGES_REQUESTED** [review]
Generally looks great, thanks! Just minor adjustment needed for temperature handling.

File: index.js (hunk 1 above)
Comment (KEGustafsson) [reply index.js:79]: Changed to Kelvin and added meta data too.
Could we keep this under `networking.modem.temperature`, then all modem data is nicely grouped in Data Browser?

Comment (bergie) [reply index.js:79]: Thanks! We're currently under way. I'll give it another look, and hopefully merge and release when we're in harbour again

**Pull request merged**